### PR TITLE
Portfolio - mobile menu

### PR DESCRIPTION
**Pull request by Kayonga99**
## In this milestone the followind were implemented:

- No linter errors.
- When the user clicks (or taps) the hamburger button, the mobile menu appears.
- When the user clicks (or taps) the close (X) button, the mobile menu disappears.
- When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
- When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+const closeMenu = document.querySelector('.close');
+const popUp = document.querySelector('.menu-up');
+
+bars.addEventListener('click', () => {
+  popUp.classList.add('pop-up');
+});
+
+closeMenu.addEventListener('click', () => {
+  popUp.classList.remove('pop-up');
+});
```
Comment (thecodechaser) [inline main.js]: - [ ] Good job on the functionality but there is a little issue according to the requirements, When the user clicks any of the mobile menu options, the mobile menu should disappear. In your project when I click on any of the mobile menu options the mobile menu doesn't disappear. Please fix this issue
  - To fix this you can add events in your javascript file, When any of the mobile menu options is clicked you can set mobile menu `display: none;`

### See the video for reference 

https://user-images.githubusercontent.com/83803414/161627952-7ce84874-d31f-4640-a0df-4ae089260f98.mp4

File: index.html [hunk 2]
```diff
@@ -8,16 +8,19 @@
...
     <a href="#" class="logo">my logo</a>
-    <nav class="nav"><img src="./photos/Union.png" alt="menu button"></nav>
-    <ul class="nav-list">
-      <li> <a href="#portifolio">Portifolio</a></li>
-      <li> <a href="#about">About</a></li>
-      <li> <a href="#contact">Contact</a></li>
-    </ul>
+    <nav class="nav"><img id='pop' src="./photos/Union.png" alt="menu button"></nav>
+    <ul class="menu-up nav-list">
+        <li class="close">&times;</li>
```
Comment (thecodechaser) [inline index.html:16]: - [ ] Good job with the styles but for this project you are supposed to stick to the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) design as much as possible. Please use the correct color and size for your close(X) button and make sure it matches the design on the template.

### Your design 

image

### Figma design

image

File: index.html [hunk 3]
```diff
@@ -8,16 +8,19 @@
   <title>My portfolio</title>
   <link rel="stylesheet" href="style.css">
 </head>
-
 <body>
   <header class="header">
     <a href="#" class="logo">my logo</a>
```
Comment (thecodechaser) [inline index.html:13]: - [ ] Please add a blur effect and decrease the color transparency for the mobile menu background and make sure it matches the design on the template.
  -To add blur you can use this `backdrop-filter: blur(5px);`

### Your design

image

### Figma design

image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @Kayonga99👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
- [ ] This project should be done in pair programming. That means that each group member needs to contribute to your project. I'm requesting some changes please make sure you implement these changes with your group(We will check the commit history to make sure that everybody has contributed code on this project).

- [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (mateo951) [reply index.html:13]: Addressed ✔

**Review by mateo951 — APPROVED** [review]
### Hi @Kayonga99, 

Well done implementing all the changes requested by your previous reviewer 🤓
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔥 
### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**